src/inference/src/dev/core_impl.cpp

```diff
@@ -1401,13 +1401,14 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::load_model_from_cache(
     ov::Plugin& plugin,
     const ov::AnyMap& config,
     const ov::SoPtr<ov::IRemoteContext>& context,
-    std::function<ov::SoPtr<ov::ICompiledModel>()> compile_model_lambda) {
+    std::function<ov::SoPtr<ov::ICompiledModel>()> compile_model_lambda,
+    bool enable_mmap) {
     ov::SoPtr<ov::ICompiledModel> compiled_model;
     struct HeaderException {};
 
     OPENVINO_ASSERT(cacheContent.cacheManager != nullptr);
     try {
-        cacheContent.cacheManager->read_cache_entry(cacheContent.blobId, [&](std::istream& networkStream) {
+        cacheContent.cacheManager->read_cache_entry(cacheContent.blobId, enable_mmap, [&](std::istream& networkStream) {
```

Comment:
```suggestion
        cacheContent.cacheManager->read_cache_entry(cacheContent.blobId, coreConfig.get_enable_mmap(), [&](std::istream& networkStream) {
```

The additional parameter `enable_mmap` can be removed as this member can access core configuration.

Reply:
Unfortunately no, it cannot. load_model_from_cache is a static method and we cannot get coreConfig.
But why is it static? 
@ilya-lavrenov @praasz. This method use CoreImpl internal state. So can we make it not static? Is there any objections to it?

Reply:
I changed it to non static const method. Please comment if it not ok

Reply:
The `core_impl` is internal implementation. There shouldn't be any issue to adjust it to what is required.